community/libmustache/APKBUILD

```diff
@@ -0,0 +1,39 @@
+# Contributor: Renoir Boulanger <hello@renoirboulanger.com>
+# Maintainer: John Boehr <jbboehr@gmail.com>
+pkgname=libmustache
+pkgver=0.4.2
+pkgrel=1
+pkgdesc="C++ implementation of Mustache intended mainly for use as a PHP extension"
+url="https://github.com/jbboehr/libmustache"
+arch="all"
+license="MIT"
+depends="libgcc libstdc++"
+makedepends="autoconf automake libtool git"
+subpackages=""
+source="$pkgname-$pkgver.tar.gz::https://github.com/jbboehr/$pkgname/archive/v$pkgver.tar.gz"
+builddir="$srcdir/$pkgname-$pkgver"
+
+prepare() {
+	cd "$builddir"
+	git clone https://github.com/jbboehr/mustache-spec.git spec && \
```

Comment:
Please do not do it this way, it requires git and prevents APKBUILD checksumming.
A better way is to add the relevant moustache-spec revision to `source=` and then symlink from the build directory, a little bit like this:

```sh
_spec_rev=b96be9fd4c6d6984828d93169fe7e86d8a8aec2f   # keep in sync with upstream!
source="$pkgname-$pkgver.tar.gz::https://github.com/jbboehr/$pkgname/archive/v$pkgver.tar.gz
	moustache-spec-$_spec_rev::https://github.com/jbboehr/mustache-spec/archive/$_spec_rev.tar.gz"

...

prepare() {
	default_prepare
	ln -s "$srcdir"/moustache-spec-$_spec_rev "$builddir"/spec
	cd "$builddir"
	autoreconf -fvi
}
```